metaflow/plugins/argo/argo_inner_flow.py

```diff
@@ -0,0 +1,39 @@
+# from metaflow import Run, namespace
+from metaflow.plugins.argo.argo_client import ArgoClient
+import time
+
+
+class TriggeredRun:
+    def __init__(
+        self,
+        flow_name: str = None,
+        parameters: dict = None,
+        wait_to_trigger: int = 100,  # In minutes (REMEMBER TO CHANGE IT TO MINUTES FOR ACTUAL VERSION b/c testing uses shorter times)
+    ):
+        from metaflow import (
+            Run,
+            namespace,
+        )  # consider cleaning this up so that import is on top
+
+        # current issue is that import on top creates circular error
+        self._flow_name = flow_name
+        self._template_name = flow_name.lower()
+
+        if parameters is None:
+            parameters = {}
+        self._parameters = parameters
+
+        self._argo_client = ArgoClient("aip-example-sandbox")
+
+        self._flow_information = self._argo_client.trigger_workflow_template(
+            self._template_name,
+            parameters=self._parameters,
+        )
```

Comment:
Just a reminder to https://github.com/zillow/metaflow/pull/181#discussion_r933490866. Feel free to create a backlog ticket to address it if you think this takes more time. `JsonType` parameter support needs to be added to test case as well.

Reply:
I added a ticket to make sure I give this issue a proper look.